markdown-link-check.json

```diff
@@ -1,13 +1,21 @@
 {
   "ignorePatterns": [
     {
-      "pattern": "^https://github.com"
+      "description": "Ignore root relative links to images which markdown-link-check is unable to verify",
+      "pattern": "^/"
```

Comment:
For example:
* https://github.com/cert-manager/website/actions/runs/3958447700/jobs/6780031828
```
FILE: content/docs/troubleshooting/webhook.md
  [✖] https://web.archive.org/web/20210903183221/https://serverfault.com/questions/1076563/creating-issuer-for-kubernetes-cert-manager-is-causing-404-and-500-error
  [✖] #io-timeout
  [✖] #context-deadline-exceeded
  [✖] #gke-private-cluster
  [✖] #diagram
  [✖] /images/troubleshooting/webhook-pod-networking-diagram.png

  48 links checked.

  ERROR: 6 dead links found!
  [✖] https://web.archive.org/web/20210903183221/https://serverfault.com/questions/1076563/creating-issuer-for-kubernetes-cert-manager-is-causing-404-and-500-error → Status: 0
  [✖] #io-timeout → Status: 404
  [✖] #context-deadline-exceeded → Status: 404
  [✖] #gke-private-cluster → Status: 404
  [✖] #diagram → Status: 404
  [✖] /images/troubleshooting/webhook-pod-networking-diagram.png → Status: 400

```